src/main/scala/org/broadinstitute/dsde/firecloud/webservice/StaticNotebooksApiService.scala

```diff
@@ -0,0 +1,26 @@
+package org.broadinstitute.dsde.firecloud.webservice
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.service._
+import org.broadinstitute.dsde.firecloud.utils.StandardUserInfoDirectives
+import spray.http.HttpMethods
+import spray.routing._
+
+trait StaticNotebooksApiService extends HttpService
+  with FireCloudDirectives
+  with StandardUserInfoDirectives {
+
+  private implicit val executionContext = actorRefFactory.dispatcher
+
+  val calhounStaticNotebooksRoot: String = FireCloudConfig.StaticNotebooks.baseUrl
+
+  val staticNotebooksRoutes: Route = {
+    path("staticNotebooks" / "convert") {
+      requireUserInfo() { _ =>
```

Comment:
Oh interesting... of course it makes sense that orch does this. Calhoun also checks auth against sam itself because it was designed to not sit behind orch. I think this is okay, but thought I'd mention it.